src/Utils/ProtectedRoutes.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { Redirect } from "react-router-dom";
+
+interface ProtectedRoutesProps {
+  component: React.FunctionComponent;
+  path: string;
+}
+
+export const ProtectedRoutes: React.FC<ProtectedRoutesProps> = (props: ProtectedRoutesProps) => {
```

Comment:
Please use `function` for functions defined globally or as prototypes in classes, see https://stackoverflow.com/questions/22939130/when-should-i-use-arrow-functions-in-ecmascript-6 for more details of why.

Reply:
Even the [CRA template](https://github.com/facebook/create-react-app/pull/8412) use that these days :/